src/options.cc

```diff
@@ -80,37 +81,32 @@ bool Options::loadOptionsFromStream(std::istream* config_file) {
 
     bool issetPort = false;
     for (unsigned int i =0; i < config.statements_.size(); i++) {
-        if (config.statements_[i]->tokens_[KEY] == "server") {
-            std::shared_ptr<NginxConfigStatement> temp_config = config.statements_[i];
-            for (unsigned int j = 0; j < temp_config->child_block_->statements_.size(); j++) {
-
-                // Sets port.
-                if(temp_config->child_block_->statements_[j]->tokens_[KEY] == "port") {
-                    if(issetPort) {
-                        std::cerr << "Multiple ports in config file.\n";
-                        return false;
-                    }
-                    else {
-                        issetPort = addPort(temp_config->child_block_->statements_[j]);
-                        if(!issetPort) {
-                            return false;
-                        }
-                    }
-                }
+        std::shared_ptr<NginxConfigStatement> temp_config = config.statements_[i];
 
-                // Sets modules.
-                else if (temp_config->child_block_->statements_[j]->tokens_[KEY] == "module"){
-                    if(addModule(temp_config->child_block_->statements_[j]) == false) {
-                        return false;
-                    }
-                }
-
-                // For future child blocks in config file.
-                else{
+        // Sets port.
+        if(temp_config->tokens_[KEY] == "port") {
+            if(issetPort) {
+                std::cerr << "Multiple ports in config file.\n";
+                    return false;
+            } else {
+                issetPort = addPort(temp_config);
+                if(!issetPort) {
                     return false;
                 }
             }
         }
+
+        // Sets path.
+        else if (temp_config->tokens_[KEY] == "path"){
+            if(addModule(temp_config) == false) {
+                return false;
+            }
+        }
+
+        // For future statements in config file.
+        else {
+            return false;
```

Comment:
I know you didn't add this right now, but it might be nice if we printed an error message, like
 `"unknown specifier '" << tokens_[KEY] << "'."`, so we can tell what goes wrong when parsing a bad config file.